### DISCUSSION TITLE
Why do package managers such as homebrew only include a subset of features?

### DISCUSSION THREAD

**Discussion by timtomch** [dathere/qsv#1757] [Q&A]
I'm curious as to why installing qsv via homebrew only brings in the `apply` and `luau` features. Is this a size constraint? Are the other features too hardware dependent for homebrew to sort out?

**Comment by ondohotola** [comment]
I think it has something to do with the production robot or something.

Pull the ZIP, use the one you like and then run it occasionally with -U

**Comment by jqnatividad** [comment]
@timtomch , I think it's really for the recipe maintainer to decide.

When it was first contributed two years ago - https://github.com/Homebrew/homebrew-core/commit/a200d0bf06fbc0c0b83dd4d1c51dfd49b915a69c, it only had qsvlite.

Shortly after, `apply` was added 
https://github.com/Homebrew/homebrew-core/commit/d1687074e986d7ca5806013b91af33cef9cb2771

And when `luau` was promoted as qsv's DSL - I made the case to enable the `luau` feature.
https://github.com/Homebrew/homebrew-core/commit/8c4b7b2cbb89ea84eb581e8d7ff3ff4b26071124

Homebrew is a very popular package manager. Perhaps, you can make the case to enable more features and create a PR?
